generators/server/templates/src/test/java/package/web/rest/AccountResourceIT.java.ejs

```diff
@@ -168,65 +151,53 @@ import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.
 <%_ } else { _%>
 import org.springframework.test.web.reactive.server.WebTestClient;
 <%_ } _%>
+import static com.tngtech.sstestsupport.web.rest.AccountResourceIT.TEST_USER_LOGIN;
```

Comment:
Should this line be removed?

Reply:
See above, I don't think it compiles without it, if you want to use the constant within the annotation...

Reply:
Ok